src/spnavd.c

```diff
@@ -153,6 +153,7 @@ int main(int argc, char **argv)
 	signal(SIGHUP, sig_handler);
 	signal(SIGUSR1, sig_handler);
 	signal(SIGUSR2, sig_handler);
+	signal(SIGPIPE, SIG_IGN);
```

Comment:
This has nothing to do with adding USB support to spacenavd. I already cherry-picked this commit and applied it to master.